docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -1259,11 +1263,11 @@ async function getPlatformTransaction(transactionId) {
 
 ### Full Example Implementation
 
-Stellar provides an example business server implementation for SEP-24. It's split into 2 parts: web UI, accessible for the end user and back-end implementation, used to get and push updates from/to the Anchor Platform.
+Stellar provides an example business server implementation for SEP-24. It's split into two parts: a web UI, accessible for the end user and back-end implementation, used to get and push updates from/to the Anchor Platform.
```

Comment:
It's not clear what the two parts are here. I would recommend doing something like: "It's split into two parts: 1) [insert first part here]; and 2) [insert second point here].

Reply:
1) web UI, accessible for the end user
2) back-end implementation, used to get and push updates from/to the Anchor Platform
Something like that?